File: jscomp/test/watch_test.js
'use strict';

Fs = require("fs");

function test(path) do
  Fs.watch(path, do
            recursive: true
          end).on("change", (function ($$event, string_buffer) do
            console.log(--[ tuple ]--[
                  $$event,
                  string_buffer
                ]);
            return --[ () ]--0;
          end end)).close();
  return --[ () ]--0;
end end

exports.test = test;
--[ fs Not a pure module ]--

File: jscomp/test/watch_test.lua
__console = {log = print};

Fs = require "fs";

function test(path) do
  Fs.watch(path, {
            recursive = true
          }).on("change", (function(__event, string_buffer) do
            __console.log(--[[ tuple ]]{
                  __event,
                  string_buffer
                });
            return --[[ () ]]0;
          end end)).close();
  return --[[ () ]]0;
end end

exports = {};
exports.test = test;
return exports;
--[[ fs Not a pure module ]]
